collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/FormManagementSettingsTest.java

```diff
@@ -116,22 +116,6 @@ public void whenPreviouslyDownloadedOnlyEnabled_checkingAutoDownload_downloadsUp
                 .assertAndDismissNotification("ODK Collect", "ODK auto-download results", "Success");
     }
 
-    @Test
-    public void whenGoogleDriveUsingAsServer_disablesPrefsAndOnlyAllowsManualUpdates() {
```

Comment:
It ended up being a lot simpler to move a bunch of the testing of these preferences into the fragment tests.